src/jit/hwintrinsicArm64.cpp

```diff
@@ -195,6 +187,46 @@ bool HWIntrinsicInfo::isScalarIsa(InstructionSet isa)
     }
 }
 
+//------------------------------------------------------------------------
+// addRangeCheckIfNeeded: add a GT_HW_INTRINSIC_CHK node for non-full-range imm-intrinsic
+//
+// Arguments:
+//    immOp      -- the operand of the intrinsic that points to the imm-arg
+//    max        -- maximum allowable value for the immOp
+//    mustExpand -- true if the compiler is compiling the fallback(GT_CALL) of this intrinsics
+//
+// Return Value:
+//     if necessary, add a GT_HW_INTRINSIC_CHK node which throws an ArgumentOutOfRangeException
+//     when the immOp is not in the valid range
```

Comment:
I would add ", otherwise, returns the provided immOp"

Reply:
Sure.